src/components/CaptureDetailDialog.js

```diff
@@ -323,10 +323,10 @@ function CaptureDetailDialog({
         <Grid item className={classes.box}>
           <Typography className={classes.subtitle}>Tags</Typography>
           <Typography variant="subtitle1">Species</Typography>
-          {species && species.name ? (
+          {species && species[0]?.name ? (
```

Comment:
Has this been consciously changed from an object to an array as part of the migration?
If not, is there an error somewhere? We don't allow multiple species to be assigned to a single capture.

Reply:
I don't know, sorry I didn't explore it but I'll check if it's coming from the api or somewhere in the client code.

Reply:
The array result is coming from the API.  It is an array now because we are using the /species endpoint with a filter for the uuid instead of making a request to /species/:id

I don't think it is worth updating the legacy API at this point.  We can make the adjustment to the client when we get the new API.

Reply:
Let's hide that workaround in the `getSpeciesById()` method, so that callers still receive a single `species` as a result.
It's fair to assume we'll have 0 or 1 items in the response from the API.